project/dependencies.scala

```diff
@@ -1,7 +1,7 @@
 import sbt._
 
 object Dependencies {
-  val capiVersion = "19.0.3"
+  val capiVersion = "19.0.5-SNAPSHOT"
```

Comment:
Will update with the right version after this gets merged: https://github.com/guardian/content-api-scala-client/pull/371